### PR TITLE
A remote month

### PR DESCRIPTION
**Pull request by DuncanFenning**
### Blog title

A Remote month (suggestions for a better title welcome)

### Blog description

My initial thoughts after transitioning to a full time remote home worker. I'm anticipating a series of posts about my move, this is the first

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2016-09-23-a-remote-month.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### Your day starts when you wake up
+
+When commuting to the shared office, I would wake up at 6:30am to start work at 9am. This may seem like a long time, but by the time I had properly woken up, showered, made lunch, had breakfast and got through rush hour traffic, that was my morning. However that has all changed. I get an extra hour in bed, awesome! I start work an hour earlier, amazing! I finish at 4pm and am already at home, incredible! I save 3 hours a day compared to commuting to the office. That's 15 hours a week, 60 hours a month. Think of all the time for activities!
+
+### No commute, mo' money?
+
+I took my commute for granted. I never considered what my commute was doing to me. It was a necessity that came with the job. Retrospectively, I was spending 2 hours a day and £40 a week getting to work. Have you ever considered spending £40 a month on something and decided it wasn't worth the money? How about £120 a month? That's how my commute felt to me. A waste of money. Now my commute is from my bed to my laptop. It's a hell of a lot cheaper and the only congestion I get stuck in is making the morning cup of coffee.
```
Comment (mattsaunby) [inline _posts/2016-09-23-a-remote-month.md]: This doesn't quite read right to me, I think it ought to either be "Have you ever considered spending £40 a month on something and deciding it isn't worth the money?" or Have you ever spent £40 a month on something and decided it wasn't worth the money?"

Comment (pmcnr-hx) [inline _posts/2016-09-23-a-remote-month.md]: Shouldn't it be 4 x £40 = £160?

**Comment by mattsaunby** [comment]
:+1: from me dude :)

File: _posts/2016-09-23-a-remote-month.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Communication
+
+Since moving away, good communication with my team has become paramount. Natural communication occurs a lot more freely when working with someone in person, so ensuring this happens when working remotely has become both mine and my team's responsibilities. I make an effort to communicate often with them and they do the same for me. Working remotely hasn't caused any issues in terms of breakdown in communication but I have to be more aware of keeping my team up to date.
+
+### So far so good!
+
+So far my remote working experience has been very positive. The loneliness hasn't kicked in yet, but I'm prepared to combat this when the time comes. The one thing I do miss is the social aspect of working in an open office. Not being able to get involved in the office BBQs will take some getting used to, but the positives definitely outweigh the negatives.
```
Comment (pmcnr-hx) [inline _posts/2016-09-23-a-remote-month.md]: Been doing the _1 month in Portugal_ remote experience and I guess, the social aspect, water cooler conversation, and such is what I miss the most also. There's that nagging feeling that you're not as much "into things" as someone who is physically in the office.

File: _posts/2016-09-23-a-remote-month.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### No commute, mo' money?
+
+I took my commute for granted. I never considered what my commute was doing to me. It was a necessity that came with the job. Retrospectively, I was spending 2 hours a day and £40 a week getting to work. Have you ever considered spending £40 a month on something and decided it wasn't worth the money? How about £120 a month? That's how my commute felt to me. A waste of money. Now my commute is from my bed to my laptop. It's a hell of a lot cheaper and the only congestion I get stuck in is making the morning cup of coffee.
+
+### Not feeling guilty
+
+Working remotely has a negative stigma attached to it. After all, how can you be productive if you aren't in the office with everyone else? You have to let your work do the talking. If you are working at a pace your team are happy with, what's to feel guilty about? Just because you have time to put the washing on, or head to the post office during lunch, it doesn't mean you're not keeping up with your workload. If anything, you can be more productive!
```
Comment (pmcnr-hx) [inline _posts/2016-09-23-a-remote-month.md]: I'm definitely more productive at home also because it's easier to manage the distractions.

**Comment by pmcnr-hx** [comment]
I guess the theme of this post is "working mostly remotely with the occasional visit to the office", compared with previous blog posts we've had which focus more on working a few days a week from home. I feel you may have focused too much on the advantages and dismissed a bit the disadvantages. I agree with all the advantages you've mentioned: I've experienced them myself. I guess, it's probably too soon, like you mention, for you to have experienced the longer term effects of remote working, so it's fair enough if the tone is a bit on the rosy side.

File: _posts/2016-09-23-a-remote-month.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+layout: post
+title: A Remote Month
+categories: Remote Working
+author: duncan_fenning
+tags: Remote Home Office
+comments: true
+excerpt: My initial thoughts about moving to remote home working.
+---
+
+A lot has changed for me in the last 2 months. The biggest change was moving away. Away from Kent, where I've lived my whole live; away from friends and family and away from the office. I knew this was coming but had no idea what to expect. A wise man once said, 'If you don't know what to expect, don't expect anything'. So that's exactly what I did, expected nothing.
```
Comment (hpoom) [inline _posts/2016-09-23-a-remote-month.md]: "where I've lived my whole **life**" not **live**

File: _posts/2016-09-23-a-remote-month.md (hunk 1 above)
Comment (hpoom) [inline _posts/2016-09-23-a-remote-month.md]: £160 as @pmcnr-hx said

**Comment by hpoom** [comment]
Great post @DuncanFenning love the use of headings to break it up. 

Regards title, you welcomed suggestions for a better title. So, if you really are going to make a series of posts, and I really hope you do, then series work best with a series title and post title. So you get `Series: Post title`. My suggestion for this blog post would be `Remote: The Autumn Migration`, then use `Remote: ***` for future blog posts. The title needs to be punchy as it helps with social sharing.

Let is know when the post is love I would love to share it twitter!

File: _posts/2016-09-23-a-remote-month.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: A Remote Month
```
Comment (hpoom) [inline _posts/2016-09-23-a-remote-month.md]: As put in general comments I would change title to **Remote: The Autumn Migration**

File: _posts/2016-09-23-a-remote-month.md (hunk 3 above)
Comment (DuncanFenning) [inline _posts/2016-09-23-a-remote-month.md]: There is another blog post coming about remote working efficiency. Im trying not to cram too much into 1 blog post :)

File: _posts/2016-09-23-a-remote-month.md (hunk 2 above)
Comment (DuncanFenning) [inline _posts/2016-09-23-a-remote-month.md]: completely agree. The benefits of the office environment at HX are lost when working remotely. Again, some thoughts I will probably share in a future blog post :)

File: _posts/2016-09-23-a-remote-month.md (hunk 1 above)
Comment (DuncanFenning) [inline _posts/2016-09-23-a-remote-month.md]: Not sure how I can to that number :/ must have been lost in my writing

**Comment by DuncanFenning** [comment]
@hpoom and @pmcnr-hx I've made the changes you recommended. Mind taking another look? Keen to get this out the door today, so I can move on to more awesome remote blog posts

**Comment by hpoom** [comment]
Sorry for the delay @DuncanFenning 

I have cast my eyes over it and looks good to me. 👍

**Pull request merged**